bot/exts/moderation/verification.py

```diff
@@ -848,6 +848,22 @@ async def bot_check(ctx: Context) -> bool:
         else:
             return True
 
+    @command(name='verify')
+    @has_any_role(*constants.MODERATION_ROLES)
+    async def apply_developer_role(self, ctx: Context, user: discord.Member) -> None:
+        """Command for moderators to apply the Developer role to any user."""
+        log.trace(f'verify command called by {ctx.author} for {user.id}.')
+        developer_role = self.bot.get_guild(constants.Guild.id).get_role(constants.Roles.verified)
```

Comment:
This should probably use `constants.Roles.verified`, and then it can be combined with the next line.